content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -116,12 +116,12 @@ be down for the same period as the previous example (`08:29:00` to `10:21:00`,)
 The CronJob is only responsible for creating Jobs that match its schedule, and
 the Job in turn is responsible for the management of the Pods it represents.
 
-## New controller
+## New vs old controller
```

Comment:
“New” is a term that soon gets stale, so how about keeping the fragment identifier and using a different heading?
```suggestion
## Controller version {#new-controller}
```

Reply:
Fair point, updated.